src/pages/workspaces/workspace/Data.js

```diff
@@ -119,15 +119,20 @@ class WorkspaceDataContent extends Component {
       try {
         this.setState({ loading: true, refreshRequested: false })
 
-        const [workspaceAttributes, { results, resultMetadata: { unfilteredCount } }] = await Promise.all([
+        const [workspaceAttributes, entities] = await Promise.all([
           getWorkspaceAttributes(),
-          Workspaces.workspace(namespace, name)
-            .paginatedEntitiesOfType(selectedDataType, {
-              page: pageNumber, pageSize: itemsPerPage, sortField: sort.field, sortDirection: sort.direction
-            })
+          isDataModel ?
+            Workspaces.workspace(namespace, name)
+              .paginatedEntitiesOfType(selectedDataType, {
+                page: pageNumber, pageSize: itemsPerPage, sortField: sort.field, sortDirection: sort.direction
+              }) :
+            Promise.resolve()
```

Comment:
Can this also be just `undefined`? I've seen that in other places.

Reply:
If I understood @zarsky-broad correctly, it couldn't be `undefined` because later in line 133 when it grabs the value of `entities.resultMetadata.unfilteredCount` that if the `resultMetadata` was undefined, then `resultMetadata.unfilteredCount` would throw an error.

Reply:
Not quite; the condition below should prevent that. I thought I remembered that `await undefined` caused issues, but it's possible I was totally wrong. Try it out @katevoss and see if anything explodes!

Reply:
Nothing exploded!